.mergify.yml

```diff
@@ -3,7 +3,11 @@ pull_request_rules:
     conditions:
       - base=master
       - status-success="validate commits"
-      - status-success="run-checks (3.7)"
+      - status-success="check python formatting (3.6)"
+      - status-success="check python formatting (3.7)"
```

Comment:
I think these status names are now out  of date. There appears to be only one format status `python format (3.7)`

To check the status names, look at the list of names on the main PR page under `Show all checks`.

You may also want to add the `coverage` check as a required status.